src/components/Character/Character.js

```diff
@@ -2,12 +2,13 @@ import React from 'react';
 import { withRouter } from 'react-router-dom';
 import PropTypes from 'prop-types';
 
+import getPose from '../../utils/getPose';
```

Comment:
It's not a big deal, but I think that for naming consistency, you should also rename two other helper functions or leave as it or put all of them in single file as they share same context (more about it below)
Inside `/src/utils` we have also:

```
extractQuoteIDFromPath.js
extractRoleFromPath.js
```

I understand why `getPose` is better fit, but still, those functions have very similar behaviour and share the same context which is url path. Maybe putting them in single file is better idea. Having them in one file could _possibly_ prevent such refactor, as you would see that they have the same prefix `extract-sth-fromPath.js`  cc: @mgolawski

Reply:
Seems like good idea, to move all **extractSomethingFromPath** to one file (we can call it _extractFromPath_ for example). And probably rename **getPose** to **extractPose** (ans shorten names of others by removing FromPath suffix).

Reply:
:+1: For extract keyword. We can gather all methods in for example `urlParser` service.